### PR TITLE
[ADD] New module: web_switch_lang

### PR DESCRIPTION
**Pull request by thinkopen-pt**
This module allow the user to easily toggle the UI between two languages with a single click. We've found that this significantly helps when translating content, since some users find the translation tree confusing. The module adds a button in the top bar (before the username) that highlights the current language and can be clicked to switch, refreshing the whole page.

### CODE REVIEW

File: web_switch_lang/COPYRIGHT [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- encoding: utf-8 -*-
```
Comment (elicoidal) [inline web_switch_lang/COPYRIGHT]: @  Thanks for your contribution to the OCA. 
Let me introduce to you some basic information about contribution in OCA and I will review other points in more comments inside the code..
In general, please follow the principles: 
- Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
- Please follow the OCA README template:
  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
- We recommend to include a screenshot as well to help new comers (nice to have)
- Please use simple headers in all py files following OCA templates
  https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

Can you remove this file: copyright notice are enclosed in the templates above

File: web_switch_lang/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (elicoidal) [inline web_switch_lang/__openerp__.py:4]: please add copyright notice following template

File: web_switch_lang/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    'name' : 'Fast languages switcher',
+    'version': '1.0',
```
Comment (elicoidal) [inline web_switch_lang/__openerp__.py]: version 8.0.1.0.0

File: web_switch_lang/__openerp__.py (hunk 2 above)
Comment (elicoidal) [inline web_switch_lang/__openerp__.py:4]: (valid for all py/js/xml files)

File: web_switch_lang/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    'author': 'ThinkOpen Solutions',
+    'category': 'Tools',
+    'complexity': 'easy',
```
Comment (elicoidal) [inline web_switch_lang/__openerp__.py]: remove key

File: web_switch_lang/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    'website': 'https://thinkopen.solutions',
+    'data': [
+        'views/templates.xml',
+    ],
+    'depends' : ['base', 'web'],
+    'qweb': ['static/src/xml/*.xml'],
+    'application': False,
+    'installable': True,
+    'auto_install': False,
+}
```
Comment (elicoidal) [inline web_switch_lang/__openerp__.py:18]: remove the image file below as well (irrelevant)

File: web_switch_lang/COPYRIGHT (hunk 1 above)
Comment (elicoidal) [inline web_switch_lang/COPYRIGHT]: Please add a README file (following template)

**Comment by dreispt** [comment]
@andreparames Please also check the lint issues on the TravisCI log.

**Comment by dreispt** [comment]
:+1:  although it would be nice to have a screenshot in the README.

**Comment by pedrobaeza** [comment]
For translating content, I prefer https://github.com/OCA/web/tree/8.0/web_translate_dialog

File: web_switch_lang/README.rst [hunk 6]
```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (elicoidal) [inline web_switch_lang/README.rst:1]: @thinkopen-pt  Thanks for your contribution to the OCA. 
Let me introduce to you some basic information about contribution in OCA and I will review other points in more comments inside the code..
In general, please follow the principles: 
- Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
- Please follow the OCA README template:
  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
- We recommend to include a screenshot as well to help new comers (nice to have)
- Please use simple headers in all py files following OCA templates
  https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: web_switch_lang/README.rst [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Web Switch Lang
```
Comment (elicoidal) [inline web_switch_lang/README.rst:6]: "Web Lang Switch"?
Besides lines above and under must be same length as title

File: web_switch_lang/README.rst [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
```
Comment (elicoidal) [inline web_switch_lang/README.rst:37]: If you do not use the standard OCA icon, you can remove this line

File: web_switch_lang/__openerp__.py [hunk 9]
```diff
@@ -1,14 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ThinkOpen Solutions
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
-    'name' : 'Fast languages switcher',
-    'version': '1.0',
-    'author': 'ThinkOpen Solutions',
-    'category': 'Tools',
-    'complexity': 'easy',
+    'name': 'Fast languages switcher',
```
Comment (elicoidal) [inline web_switch_lang/__openerp__.py:5]: Please be consistent with the README file

**Comment by elicoidal** [comment]
Please check runbot (I launched a rebuild now).
Small details but other than that :+1:

**Comment by thinkopen-pt** [comment]
Hi, we want to thank @dreispt, @pedrobaeza and especially @elicoidal for your work in reviewing this module. Unfortunately, due to an unforeseen policy change, we will have to withdraw this PR, but the advise will be very useful for our future contributions to OCA, which will continue. Thanks.

**Pull request closed without merging**